scripts/import_templates_gen3.py

```diff
@@ -52,8 +52,8 @@
 
 def _make_parser():
     parser = argparse.ArgumentParser()
-    parser.add_argument("-b", dest="src_dir", default="/repo/main",
-                        help="Repo to import from, defaults to '/repo/main'.")
+    parser.add_argument("-b", dest="src_dir", default="/sdf/group/rubin/repo/main",
+                        help="Repo to import from, defaults to '/sdf/group/rubin/repo/main'.")
```

Comment:
Please include this change in [`ap_verify_dataset_template`](https://github.com/lsst-dm/ap_verify_dataset_template/blob/main/scripts/import_templates.py#L62-L63) as well.